src/pages/jihong/Login/Login.js

```diff
@@ -1,8 +1,102 @@
 import React from 'react';
-import './Login';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+import '../../../../src/reset.scss';
+import { useState } from 'react';
 
 const Login = () => {
-  return <div></div>;
+  const [idInput, setIdInput] = useState('');
+  const [pwInput, setPwInput] = useState('');
+
+  const handleIdInput = event => {
+    setIdInput(event.target.value);
+  };
+
+  const handlePwInput = event => {
+    setPwInput(event.target.value);
+  };
+
+  const [isActive, setIsActive] = useState(false);
+
+  const isPassedLogin = () => {
+    return idInput.includes('@') && pwInput.length > 4
+      ? setIsActive(true)
+      : setIsActive(false);
+  };
+
+  const handleKeyPress = e => {
+    if (e.key === 'Enter') {
+      goToMain();
+    }
+  };
+
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/main-jihong');
+  };
+
+  const onClick = () => {
+    goToMain();
+  };
+
+  return (
+    <div className="loginContainer">
+      <form className="loginForm">
+        <h1 className="formTitle">Westagram</h1>
+
+        <form className="loginIdPwBtn">
+          <input
+            id="inputId"
+            type="text"
+            placeholder="Phone number, username or email"
+            onChange={handleIdInput}
+            onKeyUp={isPassedLogin}
+          />
+
+          <input
+            id="inputPw"
+            type="password"
+            placeholder="Password"
+            onChange={handlePwInput}
+            onKeyUp={isPassedLogin}
+            onKeyPress={
+              idInput.includes('@') && pwInput.length > 4
+                ? handleKeyPress
+                : null
+            }
+          />
+
+          <button
+            id={isActive ? 'loginBtnActive' : 'loginBtnDisable'}
+            onClick={onClick}
+            disabled={
+              !idInput.includes('@') || pwInput.length < 4 ? true : false
```

Comment:
위 선언한 isLoginValid 활용
```suggestion
            !isLoginValid
```